bp384/src/arithmetic/scalar.rs

```diff
@@ -212,19 +212,26 @@ impl Scalar {
         res
     }
 
-    /// Returns the square root of self mod n, or `None` if no square root
-    /// exists.
+    /// Atkin algorithm for q mod 8 = 5
+    /// <https://eips.ethereum.org/assets/eip-3068/2012-685_Square_Root_Even_Ext.pdf>
+    /// (page 10, algorithm 3)
     pub fn sqrt(&self) -> CtOption<Self> {
-        // Because n ≡ 3 mod 4 for brainpoolP384's scalar field modulus, sqrt
-        // can be implemented with only one exponentiation via the computation
-        // of self^((n + 1) // 4) (mod n).
```

Comment:
Oops, this should've been `todo!()`. Anyway, thanks for fixing it!

Reply:
You're welcome 👍